h/tasks/annotations.py

```diff
@@ -6,7 +6,7 @@
 
 
 @celery.task
-def fill_annotation_slim(batch_size=1000):
+def fill_annotation_slim(batch_size=1000, since="2013-01-01", until="2017-12-31"):
```

Comment:
What's the reasoning for those two default values specifically?

Reply:
2012 is the date of the first annos.

The second one is far ahead in the future to have around 2M annotations to work on. We'll have to move this window over time if this approach works.